node/src/BaseClient.ts

```diff
@@ -5665,11 +5667,14 @@ export class BaseClient {
      * ```
      */
     public async fcall(
-        func: string,
-        keys: string[],
-        args: string[],
-    ): Promise<string> {
-        return this.createWritePromise(createFCall(func, keys, args));
+        func: GlideString,
+        keys: GlideString[],
+        args: GlideString[],
+        decoder?: Decoder,
+    ): Promise<GlideString> {
```

Comment:
Please update for all `fcall` and `fcall_ro` calls, ensure that you import it from the `BaseClient`.
```suggestion
    ): Promise<ReturnType> {
```